README.md

```diff
@@ -46,6 +46,20 @@ commits are being made against alphagov without protection so
 hopefully in time we can actually report by commit with another 
 hook.
 
+## Ignoring secrets
+1. In-line `# pragma: allowlist secret` In any language's comment format.
+1. When running detect-secrets, with `detect-secrets scan --update .secrets.baseline --exclude-files go.sum`
+1. Add to your repository's `.pre-commit-config.yaml`, eg:
+```
+$ cat .pre-commit-config.yaml
```

Comment:
I don't think you need to use cat here, you can just show the name and contents of the file. However, this *is* a good candidate for a UUOC award. :joy_cat:

Reply:
TIL about the UUOC award! :laughing:

Reply:
it's copied and pasted from the official docs, I'm innocent!